### ISSUE TITLE
are we GDPR compilant?

### ISSUE THREAD

**Issue by Br3nda** [Growstuff/growstuff#2023]
I don't do much international work, so I don't now - should we be adding those annoying messages about do you really want a cookie... ?

**Comment by pozorvlak** [comment]
We probably should, yeah - @cesy?

**Comment by Br3nda** [comment]
I'm going through the advice here
https://www.lawsociety.org.nz/practice-resources/practice-areas/privacy/gdpr-compliance-in-four-steps

I think we need to look into what web analytics info ends up in new relic and scout, and recieve consent from the user

**Comment by cesy** [comment]
Yeah, we should also work out what personal data we store and make sure
that someone can request it all, as well as deleting it when they delete
their account.

**Comment by Br3nda** [comment]
right now we only mark a user as deleted . We keep their data, which they released as Creative Commons when they recorded it.

So, GDPR would trump copyright law, when they released their data?

**Comment by Br3nda** [comment]
We'll need clarity on how much of their data we must delete, given they published it publicly under an open license. We can make that clearer to a user that they are licensing their data under cc.

There is a gem for data export. 
https://rubygems.org/gems/rails-gdpr-export

And about three good candidates for the cookie/tracking acceptance

**Comment by cesy** [comment]
Stuff they released under CC is fine, it's just the rest of anything
personal.

**Comment by CloCkWeRX** [comment]
Probably want to adjust the footer links; all lead to dead ends: https://www.growstuff.org/about/contact for example.

![image](https://user-images.githubusercontent.com/365751/60702064-e9bf6700-9f3c-11e9-8617-46b75a83cf88.png)
